core/frontend/vitest.config.mts

```diff
@@ -0,0 +1,68 @@
+import { coverageConfigDefaults, defineConfig } from 'vitest/config';
+import { viteStaticCopy } from 'vite-plugin-static-copy';
+
+export default defineConfig({
+  test: {
+    dir: "src",
+    setupFiles: "./src/test/setupTests.ts",
+    // include: ["**/<testfileName>.test.ts"], // to adhere to chai's describe.only/it.only
+    browser: {
+      provider: "playwright",
+      enabled: true,
+      name: "chromium",
+      headless: true,
+      screenshotFailures: false
+    },
+    coverage: {
+      provider: "v8",
+      include: [
+        "src/**/*"
+      ],
+      exclude: [
+        ...coverageConfigDefaults.exclude,
+        "src/test/**/*",
+        "**/*.d.ts",
+        "**/*.d.tsx"
+      ],
+      reporter: [
+        "text-summary",
+        "lcov",
+        "cobertura"
+      ],
+      reportsDirectory: "./lib/cjs/test/coverage",
+    },
+    pool: "threads",
+    fileParallelism: false // Had to disable parallel test runs due to Worker related tests timing out and not fetching properly.
+  },
+  plugins: [
+    viteStaticCopy({
+      targets: [
+        {
+          src: 'lib/test/test-worker.js',
+          dest: '.'
+        },
+        {
+          src: 'lib/public/*',
+          dest: '.'
+        },
+        {
+          src: 'src/test/public/*',
+          dest: '.'
+        }
+      ]
+    })
+  ],
+  resolve: {
+    alias: {
+      "../../package.json": "../package.json",
+    }
+  },
+  optimizeDeps: {
+    include: ['@itwin/core-frontend', "@itwin/core-common", "@itwin/core-bentley"],
```

Comment:
?
are these still required?
also, why would core-frontend be explicitly imported in the tests and not imported via relative paths?

Reply:
So I removed core frontend from the include array, but I actually added geometry, orbitgt, quantity to this array. Reason is performance - omitting optimizeDeps entirely would have the tests run on avg 150s. Including these 3 modules cut the time down to 86s. I then added the remaining packages that are linked within the monorepo, and avg time got down between 72-73.
Here's vite's section on [pre-bundling and dep optimization](https://vite.dev/guide/dep-pre-bundling.html#monorepos-and-linked-dependencies), it explains the value of including packages that are linked via monorepo. As for why explicitly declaring this improves performance, vite's implicit ESM module detection might not be picking up our packages, maybe it's looking for type: "module" in package.json or the like